util/file_reader_writer.cc

```diff
@@ -743,14 +744,167 @@ class ReadaheadRandomAccessFile : public RandomAccessFile {
     return s;
   }
 
-  std::unique_ptr<RandomAccessFile> file_;
+  const std::unique_ptr<RandomAccessFile> file_;
   const size_t alignment_;
-  size_t               readahead_size_;
+  const size_t readahead_size_;
 
   mutable std::mutex lock_;
+  // The buffer storing the prefetched data
   mutable AlignedBuffer buffer_;
+  // The offset in file_, corresponding to data stored in buffer_
   mutable uint64_t buffer_offset_;
 };
+
+class ReadaheadSequentialFile : public SequentialFile {
```

Comment:
It's better to have a comment to this class explaining what the class is for.

Reply:
ok